pkg/upgrade/machine_creator.go

```diff
@@ -29,11 +31,23 @@ type nodeLister interface {
 }
 
 type machineGetter interface {
-	Get(string, metav1.GetOptions) (*clusterapiv1alpha1.Machine, error)
+	Get(name string, namespace string) (*clusterapiv1alpha2.Machine, error)
+}
+
+type GetMachine struct {
```

Comment:
All exported types need a godoc

Reply:
I'm confused why you needed to add this type? We already have the ability to fake/mock out a `machineGetter`.

Reply:
Can you remove this type?